mobile/Makefile

```diff
@@ -29,15 +30,16 @@ ifeq ($(OS),android)
 		HOST_OS="$(HOST_OS)" \
 		GO_GENERATE_CMD="go generate" \
 		SHELL=/bin/sh
+		@cp ../vendor/nim-sds/build/libsds$(LIB_EXT) $(LIB_PATH)/libsds$(LIB_EXT)
 else ifeq ($(OS),ios)
 	$(MAKE) -C ../vendor/status-go statusgo-ios-library \
 		ARCH=$(ARCH) \
 		IPHONE_SDK="$(IPHONE_SDK)" \
 		IOS_TARGET="$(IOS_TARGET)" \
 		SHELL=/bin/sh
+		@cp $(WORKSPACE)/nim-sds/build/libsds$(LIB_EXT) $(LIB_PATH)/libsds$(LIB_EXT)
```

Comment:
What is the `WORKSPACE` variable you're referencing? The one defined in Jenkins? And how to build it locally then?

Reply:
We shouldn't reference pure Jenkins variables here. Instead, we should use `NIM_SDS_LIB_DIR`.